boards/px4/fmu-v4/init/rc.board_mavlink

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+#
+# PX4 FMUv4 specific board MAVLink startup script.
+#------------------------------------------------------------------------------
+
+# Start MAVLink on the USB port
+mavlink start -d /dev/ttyACM0
+
+# Pixracer: start MAVLink on Wifi (ESP8266 port)
+mavlink start -r 20000 -b 921600 -d /dev/ttyS0
```

Comment:
Did you try configuring this via parameters? https://github.com/PX4/PX4-Autopilot/blob/cbc37f9fcda4a3f01112f542c3940dbe3ea46ea1/boards/px4/fmu-v4/default.px4board#L7

Maybe we need to be setting appropriate default parameter values instead.

Reply:
Not sure if I understand, do you mean something like this:
```suggestion
mavlink start -r 20000 -b 921600 -d {CONFIG_BOARD_SERIAL_WIFI}
```
I did not find this port being defined as a parameter, but I could add it